xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/LiveTableEditSheet.xml

```diff
@@ -150,8 +150,8 @@
     &lt;/dt&gt;
     &lt;dd&gt;#displayPropertyValue('description')&lt;/dd&gt;
     &lt;dt&gt;
-      &lt;label for="AppWithinMinutes.LiveTableClass_0_columns"&gt;$doc.displayPrettyName('columns', false, false)&lt;/label&gt;
-      &lt;span class="xHint"&gt;$services.localization.render('platform.appwithinminutes.liveTableEditorColumnsHint')&lt;/span&gt;
+      &lt;label for="availableColumns"&gt;$doc.displayPrettyName('columns', false, false)&lt;/label&gt;
+      &lt;span id='availableColumnsDescription' class="xHint"&gt;$services.localization.render('platform.appwithinminutes.liveTableEditorColumnsHint')&lt;/span&gt;
```

Comment:
As discussed on another PR, I would rather use `Hint` instead of `Description` as this is the term we use here (the translation key is also named `liveTableEditorColumnsHint`) and switching between the two just creates a mess.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2271/commits/81b4383343e9ab7846aca91b62ef38f62dd7f662 :+1: